### PR TITLE
[Ready for review] Update main Docs readme and update code comment

### PR DESCRIPTION
**Pull request by bernars-usa**
This PR fixes a couple of small things:

- It updates links in the main Docs readme according to the current reality, which is something I overlooked before merging #167. 😅 
- It takes a stab at a more helpful code comment about the `keepInPageOnReview` property (based on discussion in https://github.com/usds/us-forms-system/pull/167#discussion_r203408856).

### CODE REVIEW

File: docs/building-a-form/about-the-schema-and-uischema-objects.md [hunk 1]
```diff
@@ -230,8 +230,10 @@ The us-forms-system code includes additional `uiSchema` functionality not found
       };
     },
 
-    // Use this when you have an array field that should not be pulled out of the page its
-    // in and shown separately on the review page.
+    // By default, array fields that are displayed on a single page in a form, such as
+    // information for multiple dependents, are displayed as separate pages or sections on
```
Comment (annekainicUSDS) [inline docs/building-a-form/about-the-schema-and-uischema-objects.md]: Really nitpicky, but I might reword this to "are display in a separate section on the review page." Only because the way it works is all the info in the array field is displayed together, and the wording now makes it sound like each item within the array gets its own section, so just wanted to clarify that a bit!

Comment (bernars-usa) [reply docs/building-a-form/about-the-schema-and-uischema-objects.md]: Does that mean this:

> To keep the information in a single section on a review page, set this property.

... should be something more like:

"To keep all information together on a review page, set this property."

I'm having a hard time deciphering the difference between when the property is set and not set according to your description.

Comment (annekainicUSDS) [reply docs/building-a-form/about-the-schema-and-uischema-objects.md]: Sorry that wasn't clear. I would keep the wording "To keep information in a single section on a review page, set this property to `true`", just adding "to `true`" at the end. I think that language is the most specific about what's going on. The property only needs to be set to `true` if you want to keep the form fields together in one section; otherwise you don't need to set it all (setting it to `false` is the same as not setting it at all).

Comment (bernars-usa) [reply docs/building-a-form/about-the-schema-and-uischema-objects.md]: Got it! Thanks for clarifying.
